src/Exception/SilencedErrorException.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sentry\Exception;
+
+class SilencedErrorException extends \ErrorException
```

Comment:
Missing the DocBlock describing what this class does. Also does it makes sense to make it `final`?